src/widgets/grid.ts

```diff
@@ -1,9 +1,15 @@
+import { ElementArrayFinder, ElementFinder } from "../wdio";
 import { ContextMenu } from './context-menu';
 import { Widget } from './widget';
-import { ElementArrayFinder, ElementFinder } from "../wdio";
 
 
 export class Grid extends Widget {
+    public async waitUntilLoaded(cssTag: string , timeout: number = 5000) {
```

Comment:
We are very interested in a generic waitUntilLoaded() for Grids (we were trying something, actually), so this would be great.

But I have a question on this implementation: if it relies on a tag externally supplied to the method, could you put an example of call?, with this tag whose absence is expected in order to consider the grid loaded.

Tnx in advance

Reply:
With the new default value for CSS tag would be easier:

`somePage.getSomeGrid().waitUntilLoaded();`